test/fixture/test.js

```diff
@@ -15,7 +15,9 @@ test.afterEach.always(async t => {
 	await t.context.app.stop();
 });
 
-const cleanLogs = x => x.filter(x => !x.includes('INFO:CONSOLE'));
+const cleanLogs = x => x.filter(x => {
+	return !x.includes('INFO:CONSOLE') && !x.includes('DnsConfig');
```

Comment:
When disconnected to the internet, the following is logged to the console:

```
[71726:0331/130538.146237:WARNING:dns_config_service_posix.cc(326)] Failed to read DnsConfig.
```

So I filtered them out in case the tests are ever run without an internet connection.